torch_xla/experimental/custom_kernel.py

```diff
@@ -1054,17 +1051,14 @@ def ragged_paged_attention(
       ],
   )
 
-  num_q_blks = ceil_div(cu_q_lens[num_seqs], num_queries_per_block)
```

Comment:
i see. You removed it because we don't need to dynamic grid anymore.

Reply:
but I'm curious if `cu_q_lens[num_seqs[0]]` would work

Reply:
It cannot work,  got this error `RuntimeError: Bad StatusOr access: INTERNAL: Mosaic failed to compile TPU kernel: Slice shape along dimension 2 must be aligned to tiling (8), but is 2.`

We can figure it out later